src/views/admin/PersonDetailsView.vue

```diff
@@ -73,6 +73,7 @@
   import type { LockUserBodyParams } from '@/api-client/generated';
   import type { TranslatedObject } from '@/types';
   import { DIN_91379A, NO_LEADING_TRAILING_SPACES } from '@/utils/validation';
+import { useConfigStore, type ConfigStore } from '@/stores/ConfigStore';
```

Comment:
Das sieht in github verrutscht aus. Bitte einrücken

Reply:
npm run lint ausgeführt! danke